lib/src/main/java/com/orange/ods/compose/component/button/OdsOutlinedButton.kt

```diff
@@ -79,7 +80,7 @@ fun OdsOutlinedButton(
             )
         ) {
             icon?.Content()
-            Text(text = text.uppercase(), style = OdsTheme.typography.labelL)
+            OdsText(text = text, style = OdsTextStyle.LabelL)
```

Comment:
I think `displaySurface` parameter is missing cause text doesn't appear when display is forced on light or dark.